CHANGELOG.md

```diff
@@ -5,6 +5,8 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.1.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+- Adding resource for users to update user details.
+- Changed the default access on the users resource to be 'member' because it's now the majority of use cases.
```

Comment:
Please add a heading for the type of changes these are, and please use imperative verbs for the changelog:

```suggestion
### Features
- Add resource for users to update user details.

### 🏡 Chore
- Change the default access on the users resource to be 'member'.
```

Reply:
It's also nice to add a PR or issue number to the end too: